### PR TITLE
Use hosted memcache alpha

### PR DESCRIPTION
**Pull request by kevmoo**
Should wait for https://github.com/dart-lang/appengine/pull/71 to land and a publish

...but wanted to show that this works

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @isoos @mkustermann

File: app/pubspec.yaml [hunk 1]
```diff
@@ -32,3 +32,9 @@ dev_dependencies:
   build_runner: ^0.4.0
   source_gen: '^0.7.0'
   test: ">=0.12.20 <0.13.0"
+
+dependency_overrides:
+  appengine:
+    git:
+      url: https://github.com/dart-lang/appengine
+      ref: memcache_fun
```
Comment (mkustermann) [inline app/pubspec.yaml]: Please don't make PRs with things that should not be committed!

**Comment by kevmoo** [comment]
> The only necessary change in order to switch to hosted memcache is the small change in Dockerfile.

Yup. Wanted to try out a few other things while hacking...

File: app/pubspec.yaml (hunk 1 above)
Comment (kevmoo) [reply app/pubspec.yaml]: See WIP in the title – hoping folks pay attention before hitting merge – it was an easy way to share and discuss the change.

**Comment by kevmoo** [comment]
@mkustermann – PTAL

**Pull request merged**